README.md

```diff
@@ -1,23 +1,95 @@
 
 # V1 suggestor
-* Reads V1 metadata from the source kafka queue
-* filters and transforms it to UP standard json representation
-* puts the result on the destination kafka queue
-
-# How to run
-
-```
-export|set SRC_ADDR=http://kafka:8080
-export|set SRC_GROUP=FooGroup
-export|set SRC_TOPIC=FooBarEvents
-export|set SRC_QUEUE=kafka
-export|set SRC_CONCURRENT_PROCESSING=true
-export|set DEST_ADDRESS=http://kafka:8080
-export|set DEST_TOPIC=DestTopic
-export|set DEST_QUEUE=kafka
+Processes metadata about content that comes from QMI system - aka V1 annotations.  
 
+* Reads V1 metadata for an article from the  kafka source topic _NativeCmsMetadataPublicationEvents_
+* Filters and transforms it to UP standard json representation
+* Puts the result onto the kafka destination topic _ConceptSuggestions_
+
+v1-suggestor service communicates with kafka via http-rest-proxy. It polls kafka-rest-proxy for messages and POSTs transfirmed messages to kafka-rest-proxy.  
+This service is deployed in the Delivery clusters.
+## Installation
+
+* `go get -u github.com/Financial-Times/v1-sugestor`
+* `cd $GOPATH/src/github.com/Financial-Times/v1-sugestor`
+* `go install`
+
+## Startup parameters
+
+| **Parameter** | **Value in prod** | **Explained** |
+|---|---|---|
+| **SRC_ADDR** |_http://localhost:8080_ | Url of the _http-rest-proxy_ host to connect to in order to **receive** messages from kafka. |
+| **SRC_GROUP** | _v1Suggestor_ | The consumer group for receiving messages from kafka. |
+| **SRC_TOPIC** | _NativeCmsMetadataPublicationEvents_ | kafka topic to consume messages from. |
+| **SRC_QUEUE** | _kafka_ |  Used by _Vulkan_ to route http requests based on _Host_ header. In docker cluster all hosts are at _http://localhost:8080_. This http header is supplied to distinguish one service from another.  Host header _kafka_ points to _http-rest-proxy_. |
+| **SRC_CONCURRENT_PROCESSING** | _false_ | Should the consumer process messages concurrently or sequentially. |
+| **DEST_ADDRESS** | _http://localhost:8080_| Url of the _http-rest-proxy_ host to connect to in order to **send** messages to kafka. In prod env this is typically the same address as the SRC_ADDR. |
+| **DEST_TOPIC** | _ConceptSuggestions_ | kafka topic to **send** messages to.  |
+| **DEST_QUEUE** | _kafka_ |  Used by _Vulkan_ to route http requests based on _Host_ header. In prod docker cluster it is the same as SRC_QUEUE. |
+
+
+## Prerequisites
+1.  In order to set LDFLAGS to provide correct versionning information to */build-info* endpoint we need to enable access to remote GIT repo via https (not ssh) because Docker does not have your ssh keys.                                                                                                                                                                     
```

Comment:
I don't think it belongs here though as for it's alternative location why not bring it up the tech forum